x-pack/plugins/actions/server/builtin_action_types/jira/api.ts

```diff
@@ -80,6 +80,9 @@ const pushToServiceHandler = async ({
   if (comments && Array.isArray(comments) && comments.length > 0) {
     res.comments = [];
     for (const currentComment of comments) {
+      if (!currentComment.comment) {
+        break;
```

Comment:
I think we should `continue` because we need the other comments to be created in the external service.

Reply:
Yeah I agree with @cnasikas here, we want to `continue` rather than `break` otherwise the remaining comments will be skipped.

Reply:
Yeah I agree with @cnasikas here, we want to `continue` rather than `break` otherwise the remaining comments will be skipped.